README.md

```diff
@@ -20,6 +20,16 @@ Then, update your dependencies:
 $ mix deps.get
 ```
 
+Make sure to tell Hunter which API client to use in your config:
```

Comment:
@optikfluffel After e3ed287 we don't need this information in the `README`, but could be useful in the `Hunter.Config` module, wdyt?